pallas-math/src/math_gmp.rs

```diff
@@ -279,6 +317,18 @@ impl FixedPrecision for Decimal {
             )
         }
     }
+
+    fn round(&self) -> Self {
+        todo!()
+    }
+
+    fn floor(&self) -> Self {
+        todo!()
+    }
+
+    fn ceil(&self) -> Self {
+        todo!()
+    }
```

Comment:
are you leaving the TODOs for another PR?

Reply:
math_gmp.rs is going to be removed before I merge. Too many headaches with it.

Reply:
It's gone.